addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -82,52 +82,59 @@ export class MassMailingHtmlField extends HtmlField {
             return super.commitChanges();
         }
         if (!this._isDirty()) {
-            return;
+            // In case there is still a pending change while committing the
+            // changes from the save button, we need to wait for the previous
+            // operation to finish, otherwise the "inline field" of the mass
+            // mailing might not be saved.
+            return this._previousCommitChange;
```

Comment:
Shouldn't this variable be reset at some point ? Like when the conversion is over ? I feel like there might be a scenario where you end up with a previous promise set in this property even though it is unrelated to your current situation. You'd then return the wrong promise if you somehow manage to reach the code here without redefining `this._previousCommitChange`.